lib/evss/aws_creds.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require 'net/http'
+
+module EVSS
+  class AwsCreds
+    METADATA_ENDPOINT = 'http://169.254.169.254/latest/meta-data/iam/security-credentials/CloudWatchRole/'
+
+    def self.fetch(fetch_from_remote = false)
```

Comment:
I think @aub might disagree with you on this one given his current Rubocop PR 😄  I am indifferent on that, the variable/method name could probably be changed as there is no other source of data other than the instance metadata endpoint